README.md

```diff
@@ -1,27 +1,25 @@
-# Project Name
+# Project Rock Paper Scissors
 
-> One paragraph statement about the project.
+> Rock Paper Scissors game webpage
 
 ![screenshot](./app_screenshot.png)
```

Comment:
https://ui.dev/amiresponsive Take a look at this, your project screenshot should look a lot like that, it will display for mobile, tab, desktop and monitor, you will just scroll down the page and see a point where you will add you netlify deploy link and see the outcome you will see something like this
![image](https://github.com/Yinkusadel/odin-project-rock-paper-scissors/assets/111710949/140e89eb-3b2f-4e9d-9cac-72feeecdf042)